### PR TITLE
[DOCS] Update ES quick start for security ON by default

### PR DESCRIPTION
**Pull request by lockewritesdocs**
Updates the Elasticsearch quickstart guide to reflect behavior with security ON by default.

Preview link: https://elasticsearch_80735.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/getting-started.html

Click the **Self-managed** tab to view changes.

### CODE REVIEW

**Comment by lockewritesdocs** [comment]
@elasticmachine update branch

**Comment by lockewritesdocs** [comment]
@elasticmachine run docs

File: docs/reference/tab-widgets/quick-start-install.asciidoc [hunk 1]
```diff
@@ -21,9 +21,37 @@ Desktop].
 ----
 docker network create elastic
 docker pull {docker-repo}:{version}
-docker run --name es01-test --net elastic -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" {docker-image}
+docker run --name es-node01 -p 9200:9200 -p 9300:9300 -it {docker-image}
```
Comment (jkakavas) [inline docs/reference/tab-widgets/quick-start-install.asciidoc:24]: let's leave the `--net elastic` , I think it's good practice to have a specific docker network

**Review by jkakavas — APPROVED** [review]
LGTM, feel free to merge when the comment is addressed

**Pull request merged**